eppoclient/client.go

```diff
@@ -14,6 +14,7 @@ type EppoClient struct {
 	logger          IAssignmentLogger
 }
 
+// newEppoClient is the unexported constructor function
 func newEppoClient(configRequestor iConfigRequestor, poller *poller, assignmentLogger IAssignmentLogger) *EppoClient {
```

Comment:
making it clear that this is not available publically

Reply:
nit: that's a redundant comment in Go as everything starting with lowercase letter is not exported

Reply:
You're right - thank you.